ibm-watson-workitem/src/main/java/org/jbpm/process/workitem/ibm/watson/ClassifyImageWorkitemHandler.java

```
@@ -0,0 +1,124 @@
+                    VisualRecognition service = new VisualRecognition(VisualRecognition.VERSION_DATE_2016_05_20);
+                    service.setApiKey(apiKey);
+                }
+
+                ByteArrayInputStream imageStream = new ByteArrayInputStream(classificationImage.getContent());
+
+                ClassifyOptions classifyOptions = new ClassifyOptions.Builder()
+                        .imagesFile(imageStream)
+                        .imagesFilename(classificationImage.getName())
+                        .parameters("{\"owners\": [\"me\"]}")
+                        .build();
+                ClassifiedImage result = service.classify(classifyOptions).execute().getImages().get(0);
+                if (result.getError() != null) {
+                    ErrorInfo errorInfo = result.getError();
+                    logger.error("Error classifying image: " + errorInfo.getDescription());
+                    workItemManager.abortWorkItem(workItem.getId());
+                } else {
+                    List<ImageClassificationResult> resultList = new ArrayList<>();
+                    for (ClassifierResult classification : result.getClassifiers()) {
+                        for (ClassResult classResult : classification.getClasses()) {
+                            resultList.add(new ImageClassificationResult(classification,
+                                                                         classResult));
+                        }
+
+                        widResults.put(RESULT_VALUE,
+                                       resultList);
+                    }
+                    workItemManager.completeWorkItem(workItem.getId(),
+                                                     widResults);
+                }
+            } catch (Exception e) {
+                handleException(e);
+            }
+        } else {
+            logger.error("Missing image for classification.");
+            workItemManager.abortWorkItem(workItem.getId());
```

same here, maybe throw an exception...